setup.sh

```diff
@@ -9,6 +9,8 @@ usage(){
 	echo "-b [branch]         clone specified branch (default = Run2_2017)"
 	echo "-a [protocol]       use protocol to clone (default = ssh, alternative = https)"
 	echo "-j [cores]          run CMSSW compilation on # cores (default = 8)"
+    echo "-n [name]           name of the CMSSW directory (default = CMSSW_9_4_10)"
```

Comment:
this file uses tabs for indentation rather than 4 spaces, please make it consistent

Reply:
also, define the variable CMSSWVER before this function, and use it to print the default (instead of duplicating CMSSW_9_4_10)

Reply:
Fixed

Reply:
by "make it consistent" I meant use tabs everywhere, not spaces

Reply:
Whoops. My bad. Fixed.